_posts/2021-08-23-using-lra.adoc

```diff
@@ -0,0 +1,490 @@
+---
+layout: post
+title: An Extension for Long Running Activities
```

Comment:
Is activity not already a well understood term? Activity and Action are often used interchangeably in computing and if using them that way does not hinder the users understanding of LRA then I do not see any downside to using the words interchangeably  (in fact it encourages a more expansive view of what the specification is providing to business application developers since Activity probably encompasses the definition of Action).

Reply:
Within the LRA we use the most the term `Action`. I understand it can be used interchangeably. My point was that would be good to be consistent to naming used in LRA spec.
From what you say I understand you prefer the term `Activity` here. That's fine with me.